master/internal/command/command.go

```diff
@@ -158,7 +196,12 @@ func (c *command) Receive(ctx *actor.Context) error {
 			Notebook: c.toNotebook(ctx),
 			Config:   protoutils.ToStruct(c.Config),
 		})
-
+	case *apiv1.IdleNotebookRequest:
+		if !msg.Idle {
+			now := time.Now()
+			c.lastActivity = &now
```

Comment:
fwiw, for this annoyance there is `ptrs.TimePtr()` that rb added. also, should this be called like "SetIdleness" ha.. Idk can't think of a great name for it.

Reply:
Let's make the names consistent when we refactor this. I think it's fine for now.